Doc/library/enum.rst

```diff
@@ -424,9 +424,9 @@ Data Types
       Using :class:`auto` with :class:`IntEnum` results in integers of increasing
       value, starting with ``1``.
 
-   .. versionchanged:: 3.11 :meth:`__str__` is now :func:`int.__str__` to
+   .. versionchanged:: 3.11 *__str__* is now ``int.__str__`` to
```

Comment:
I have slightly modified your proposed change, because reading 

```
:meth:`~object.__str__` is now :meth:`!int.__str__`
```

does not make much sense to me. Now it is:

```
:meth:`!__str__` is now :meth:`!int.__str__`
```

Reply:
But the latter _is_ essentially what readers will actually see, of course; the `~` removes the `object` prefix, so the original suggestion already displays as your second example, just with `__str__` usefully cross-referenced to a description of the `__str__` dunder and what it's used for, which seems to me to be potentially helpful to readers (and surely does no harm, at least).